content/en/blog/_posts/2022-10-05-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+title: "Current State: 2019 Third Party Security Audit of Kubernetes"
+date: 2022-10-05
+slug: current-state-2019-third-party-audit
+---
+
+**Authors**: Cailyn Edwards, Pushkar Joglekar, Rory McCune and Rey Lejano
```

Comment:
```suggestion
**Authors**: Cailyn Edwards (Shopify), Pushkar Joglekar (VMware), Rory McCune (DataDog) and Rey Lejano (SUSE)
```
As suggested by @sftim in this [comment](https://github.com/kubernetes/website/pull/36971#discussion_r981474676)